service.py

```diff
@@ -25,7 +25,7 @@
 #################################################################################################
 
 LOG = LazyLogger(__name__)
-DELAY = int(settings('startupDelay') if settings('SyncInstallRunDone.bool') else 4 or 0)
+DELAY = int(settings('startupDelay') if settings('SyncInstallRunDone.bool') else 4)
```

Comment:
Does `settings('startupDelay')` always return a number? This kinda looks like a fallback for None or empty string.

In any case, I'd move the `int()` to only wrap the setting
```suggestion
DELAY = int(settings('startupDelay')) if settings('SyncInstallRunDone.bool') else 4
```


```suggestion
DELAY = int(settings('startupDelay') or 0) if settings('SyncInstallRunDone.bool') else 4
```

Reply:
The startup delay is set in the addon settings and is limited to being a number